src/event.lib.php

```diff
@@ -722,4 +722,54 @@ public static function setNewConfigKeys()
 
         return $resp;
     }
+
+    /**
+     * Clear last logins or failed login logs.
+     * 
+     * This can also be done via Sucuri Security -> Settings -> Data Storage,
+     * however to improve the user experience, a button on Last Logins  and on
+     * Failed logins sections was added and it triggers the removal of
+     * sucuri/sucuri-lastlogins.php and sucuri/sucuri-failedlogins.php.
+     * 
+     * @param string $filename Name of the file to be deleted.
+     *
+     * @return HTML Message with the delete action outcome.
+     */
+    public static function clearLastLogs($filename)
+    {
+        // Get the complete path of the file.
+        $filepath = SucuriScan::dataStorePath($filename);
+
+        // Do not proceed if not possible.
+        if (
+            !is_writable(dirname($filepath)) ||
+            !file_exists($filepath) ||
+            is_dir($filepath) ||
+            empty($filepath)
```

Comment:
`empty($filename)` just checks if the string `$filename` is empty, not the file. And I don't think we care if the directory is writable (the file may be writable even if the directory isn't).